.storybook/main.ts

```diff
@@ -93,7 +97,10 @@ module.exports = {
                                 loader: "css-loader",
                                 options: {url: false},
                             },
-                            "less-loader",
+                            {
+                                loader: "less-loader",
+                                options: {lessOptions: {math: "always"}},
```

Comment:
The newer version of Less that we now use does math slightly differently, so we need to set this setting to cause it to work like the older Less.